packages/api-form-builder/src/plugins/crud/forms.crud.ts

```diff
@@ -305,15 +268,16 @@ export const createFormsCrud = (params: CreateFormsCrudParams): FormsCRUD => {
                 webinyVersion: context.WEBINY_VERSION
             };
 
+            let result;
```

Comment:
Ok, let's throw a meaningful error in case the `formStats` creation fails, something like:

```
throw new WebinyError(
    ex.message || "Could not create form.",
    ex.code || "CREATE_FORM_STATS_ERROR",
    {
        form
    }
);